bundles/core/org.eclipse.smarthome.core.audio/src/main/java/org/eclipse/smarthome/core/audio/internal/AudioConsoleCommandExtension.java

```diff
@@ -45,9 +46,9 @@ public AudioConsoleCommandExtension() {
     @Override
     public List<String> getUsages() {
         return Arrays.asList(new String[] {
-                buildCommandUsage(SUBCMD_PLAY + " <sink> <filename>",
+                buildCommandUsage(SUBCMD_PLAY + " <sink> <filename> <volume>",
```

Comment:
It has not bee really correct before (IMHO), but as you touched it anyway shouldn't it be changed to

```text
" <sink> [<filename> [<volume>]]"
```